buildpack/telemetry/metrics.py

```diff
@@ -484,21 +493,13 @@ def _get_number_of_files(self):
     def _get_size_of_files(self):
         conn = self._get_db_conn()
         with conn.cursor() as cursor:
-            try:
-                cursor.execute(
-                    "SELECT sum(size) from system$filedocument WHERE hascontents=true;"  # noqa:E501
-                )
-                rows = cursor.fetchall()
-                if len(rows) == 0:
-                    return 0
-                return int_or_default(rows[0][0])
-            except Exception:
-                # We ignore errors here, as the information is
-                # not available for older mendix versions
-                logging.debug(
-                    "METRICS: Error retrieving file sizes", exc_info=True
-                )
-                return 0
+            cursor.execute(
```

Comment:
why remove cursor.execute from the try block?

Reply:
It's not removed, the try block is just moved to L353, to be consistent with the way we get the number of files. The old logic essentially went "try to get size of files, if it fails, must be the version!". In this way it first checks the version, and then if it fails it'll throw an actual exception.